scripts/check-locales-folder.ts

```diff
@@ -0,0 +1,116 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as glob from "glob";
+
+type LocaleDataType = {
+  [key: string]: string | LocaleDataType;
+};
+
+function loadJson(filePath: string): LocaleDataType {
+  const data = fs.readFileSync(filePath, "utf-8");
+  return JSON.parse(data) as LocaleDataType;
+}
+
+function getAllKeys(obj: LocaleDataType, parentKey: string = ""): string[] {
+  let keys: string[] = [];
+  for (const key in obj) {
```

Comment:
we can iterate object keys with 
```ts
Object.keys(obj).forEach
```
in that case we don't need the following check if the key is not inherited, because Object.keys returns only object's own keys (`if (Object.prototype.hasOwnProperty.call(obj, key)) {`)